src/compile/selection/index.ts

```diff
@@ -87,8 +87,8 @@ function getFacetModel(model: Model): FacetModel {
   return parent as FacetModel;
 }
 
-export function unitName(model: Model) {
-  let name = stringValue(model.name);
+export function unitName(model: Model, escape = true) {
```

Comment:
We prefer to use objects for boolean properties to make them more understandable. 

```suggestion
export function unitName(model: Model, {escape} = {escape: true}) {
```

Reply:
Thanks for the suggestion, @domoritz! Implemented in 283419f.